region_file_updater/__init__.py

```diff
@@ -77,6 +77,15 @@ def __repr__(self):
 		return 'Region[x={}, z={}, dim={}]'.format(self.x, self.z, self.dim)
 
 
+def print_log(server: ServerInterface, msg: str):
+	server.logger.info(msg)
+	with open(LogFilePath, 'a') as logfile:
+		try:
+			logfile.write(time.strftime('%Y-%m-%d %H:%M:%S', time.localtime(time.time())) + ': ' + msg + '\n')
+		except Exception as e:
+			server.logger.info('错误写入日志文件: {}'.format(e))
```

Comment:
It's fine to use English if you are not familiar with Chinese xd. And the try-except should include the `open` file operation

Reply:
Okey srry for my bad chinese xd.
For me works like this, thats why i made it like that. But it's better as u said, I will change it now.

Reply:
u can open an issue or something for no english support xd so i might finish the translation thing when i have time

Reply:
I have already translated the whole thing to english, but in the code itself, so issue would be cool, and then make a pull request or something with /lang thing. I could make it soon, it's just make /lang for chinese and english.